catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/ValidationQueryDelegate.java

```diff
@@ -57,17 +55,20 @@ public Boolean not(Boolean operand) {
 
     @Override
     public Boolean propertyIsEqualTo(String propertyName, String literal, boolean isCaseSensitive) {
-        return propertyName.equals(VALIDATION_ERRORS) || propertyName.equals(VALIDATION_WARNINGS);
+        return propertyName.equals(BasicTypes.VALIDATION_ERRORS)
```

Comment:
These three methods (`propertyIs...`) don't make any sense to me (they all do the same thing). Is that really what we want/need? If so, maybe we should have some kind of comments to explain why they do what they're doing right now (if you don't know, that's fine, you can ignore).